content/using-atom/sections/github-package.md

```diff
@@ -85,25 +92,29 @@ Once you've staged your changes, enter a commit **message**. Feel free to descri
 If you need more space, click the expand icon at the bottom right. It will open a commit editor in the center.
 
 
-#### Amend previous commit
+#### Amend and undo
 
-In case you forgot to commit a change and would like to add it to your previous commit, clicking the "Amend" checkbox. Now you can stage more changes or adjust the commit message.
+In case you forgot to commit a change and would like to add it to your previous commit, right-click on the last commit, then choose "Amend" from the context menu.
 
 ![Amend previous commit](../../images/github-amend.png "Amend previous commit")
 
+In case you want to edit the commit message of your last commit, or add/remove changes, click on the "Undo" button. It will roll back to the state just before you clicked on the commit button.
+
+![Undo previous commit](../../images/github-undo.png "Undo previous commit")
+
 
-#### Push
+#### Publish and push
 
-When you're ready to share your changes with your team members, click the **Push** button in the Status Bar tooltip. If your local branch doesn't exist on the remote repository, Atom will offer to create a remote branch for you.
+When you're ready to share your changes with your team members, click the **Publish** button in the Status Bar. It will push your local branch to the remote repository. After making more commits, you can **Push** them as well from the Status Bar.
 
-![Push commits](../../images/github-push.png "Push commits")
+![Publish and push commits](../../images/github-publish-push.png "Publish and push commits")
 
 
-#### Pull
+#### Fetch and pull
 
-Click on the **Fetch** button to see if anyone made changes. **Pull** to get the changes into your local branch.
+From time to time it's a good idea to click on the **Fetch** button to see if any other team member pushed changes. If so, click on **Pull** to add the changes also to your local branch.
```

Comment:
"to add the changes also to your local branch" should probably be "to merge the changes into your local branch" to be nice and explicit about the action you're taking.

It might be worth calling out [the pull.rebase config setting](https://git-scm.com/docs/git-config#git-config-pullrebase) to mention how to change this... or maybe that's too much detail.

Reply:
Yeah, maybe good to mention rebase. I remember people asking about it. I added:

---

If you prefer to **rebase** when pulling, you can configure Git to make it the default behavior:

```
git config --global --bool pull.rebase true
```

Learn more about [merge vs. rebase](https://mislav.net/2013/02/merge-vs-rebase/).

---

It links to fellow Hubber `@mislav`'s blog post. :grinning: